### ISSUE TITLE
apr qa golden_output: the greeting case's "!" pattern scores degenerate output ("!!!!", token id 0) as correct

### ISSUE THREAD

**Issue by noahgift** [paiml/aprender#3782]
**`apr qa`'s golden_output gate scores degenerate output as correct on the greeting case.**

`crates/apr-cli/src/commands/golden_output.rs:122`: the greeting case's `expected_patterns` include `"!"`:
```rust
vec!["Hello", "Hi", "hey", "hello", "well", "!"],
```
The check is substring-any, so output of `!!!!!!!!…` passes the greeting case. In the Qwen vocab `!` is **token id 0**, which is exactly what a model with dead logits emits. It's also #3726's failure shape: non-ASCII bytes map to id 0.

**Measured 2026-09-21** in the CRUX inference dogfood's integration run (#3739 / #3774, lambda). A broken HF transformers load (lm_head untied) emitted `!` × 64 for every prompt. The CRUX judge, which inherits these exact patterns through its drift-checked prompt set, scored the greeting cell **correct**. It scored 2+2 and Paris correctly as wrong, because their patterns are `"4"` and `"Paris"`.

- **CRUX is closed on its side** (#3774): a degenerate answer is no answer, from any engine. Degenerate means 8 or more non-space characters with 90% or more of them one character. It has a case-table row and a mutant.
- **The Rust gate still has the hole.** An apr build that emits token 0 in a loop passes golden case 2 of `apr qa`.

## done_when
- [ ] The greeting case can no longer be satisfied by degenerate output: drop `"!"`, or require a letter-bearing pattern.
- [ ] `golden_output` rejects a degenerate completion outright (a dominant repeated character or token), on every case.
- [ ] A test feeds `!!!!!!!!` (and a token-0 loop) and gets FAIL; the mutant that restores `"!"` turns it green.
- [ ] `scripts/crux_inference_prompts.json` keeps matching (its drift check against `golden_test_cases` will require the same edit there).

**Comment by noahgift** [comment]
closed by triage (aprender-dd, #3988 FIXED row, cop OK): fixed by `f7a8e65ac`, all in v0.69.1. Evidence: a degenerate completion is not a correct answer
